jquery.pjax.js

```diff
@@ -311,7 +317,7 @@ function pjax(options) {
     //
     // http://www.w3.org/html/wg/drafts/html/master/forms.html
     var autofocusEl = context.find('input[autofocus], textarea[autofocus]').last()[0]
```

Comment:
Shouldnt we also defer this DOM operation to only evaluate when `blurFocus` is true?